docs/docusaurus/docs/core/installation_and_setup/manage_credentials.md

```diff
@@ -1,145 +1,214 @@
 ---
 title: Manage credentials
+toc_min_heading_level: 2
+toc_max_heading_level: 2
 ---
 
 import TabItem from '@theme/TabItem';
 import Tabs from '@theme/Tabs';
 
 import InProgress from '../_core_components/_in_progress.md'
 
-Many environments and data storage systems require the provision of credentials to access.  It is important for these credentials to be stored in a secure way, outside of version control.  Great Expectations (GX) offers two ways to store and access secrets out of the box: in a YAML file that exists outside of version control, or as environment variables.  GX also supports third party secrets managers for Amazon Web Services, Google Cloud Platform, and Microsoft Azure. 
+To access a deployment environment or a data storage system you must provide your access credentials. These access credentials must be stored securely outside of version control. With GX 1.0 you can store you access credentials as environment variables, in a YAML file that exists outside of version control, or in a third-party secrets manager. GX 1.0 supports the AWS Secrets Manager, Google Cloud Secret Manager, and Azure Key Vault secrets managers. 
 
-<Tabs 
-  queryString="credential-style" 
-  groupId="credentials" 
-  defaultValue='yaml' 
-  values={
-   [
-    {label: 'Environment variables', value:'environment_variables'},
-    {label: 'YAML file', value:'yaml'},
-    {label: 'Secrets manager', value:'secrets_manager'},
-   ]
-  }>
+## Environment variables
 
-  <TabItem value="yaml" label="YAML file">
-  </TabItem>
+1. To store your database password and connection string as environment variables, run ``export ENV_VAR_NAME=env_var_value`` in a terminal or add the commands to your ``~/.bashrc`` file. For example:
 
-  <TabItem value="environment_variables" label="Environment variables">
-  </TabItem>
+    ```bash title="Terminal" name="docs/docusaurus/docs/oss/guides/setup/configuring_data_contexts/how_to_configure_credentials.py export_env_vars"
+    ```
 
-  <TabItem value="secrets_manager" label="Secrets manager">
+2. Run the following code to use the `connection_string` parameter values when you add a `datasource` to a Data Context:
 
-Select one of the following secret manager applications:
+    ```python title="Python" name="docs/docusaurus/docs/oss/guides/setup/configuring_data_contexts/how_to_configure_credentials.py add_credentials_as_connection_string"
```

Comment:
The second comment in this code block has some nonstandard capitalization (`environment Variable`) that should be corrected.

Reply:
@Rachel-Reverie this has been corrected.